modules/git/blame.go

```diff
@@ -114,7 +115,7 @@ func (r *BlameReader) Close() error {
 
 // CreateBlameReader creates reader for given repository, commit and file
 func CreateBlameReader(ctx context.Context, repoPath, commitID, file string) (*BlameReader, error) {
-	return createBlameReader(ctx, repoPath, GitExecutable, "blame", commitID, "--porcelain", "--", file)
+	return createBlameReader(ctx, repoPath, setting.Git.Path, "blame", commitID, "--porcelain", "--", file)
```

Comment:
we should get the path from the localService ?

Reply:
We should move blame to use git command but currently I didn't find a better way to do that.
For further PRs, we need to move all reference of setting.RepoRootPath to one place.